src/eval-frame/actions/eval-frame-tasks.js

```diff
@@ -39,6 +39,20 @@ tasks.closePanes = new UserTask({
   },
 })
 
+tasks.increaseEditorWidth = new UserTask({
+  title: 'Set Editor Size',
+  keybindings: ['ctrl+right'],
```

Comment:
this should be `ctrl+shift+right` as well - if I am focused on the iframe I lose the keybinding